.env.development

```diff
@@ -0,0 +1,2 @@
+//.env.development
+REACT_APP_API_ENDPOINT=http://localhost:5000
```

Comment:
This defines the URL for the API endpoint on dev environments. This isn't used in production. See code below for where the production URL is being set